src/internet_identity/internet_identity.did

```diff
@@ -242,6 +242,8 @@ service : (opt InternetIdentityInit) -> {
     register : (DeviceData, ChallengeResult) -> (RegisterResponse);
     add : (UserNumber, DeviceData) -> ();
     update : (UserNumber, DeviceKey, DeviceData) -> ();
+    // Atomically replace device matching the device key with the new device data
+    swap : (UserNumber, DeviceKey, DeviceData) -> ();
```

Comment:
I think "replace" is more accurate (as also suggested by your comment here) since "swap" suggests device 1 will replace device 2 and device 2 will replace device 1